include/triggeralgs/Supernova/TriggerActivityMakerSupernova.hpp

```diff
@@ -66,18 +67,18 @@ class TriggerActivityMakerSupernova : public TriggerActivityMaker
     return ta;
   }
 
-  int64_t m_time_start = 0;
-  int64_t m_time_end = 0;
-  int64_t m_time_peak = 0;
-  int64_t m_time_activity = 0;
-  uint16_t m_channel_start = 0; // NOLINT(build/unsigned)
-  uint16_t m_channel_end = 0;   // NOLINT(build/unsigned)
-  uint16_t m_channel_peak = 0;  // NOLINT(build/unsigned)
+  timestamp_t m_time_start = 0;
```

Comment:
Instead of initializing these to a literal zero, could `define` or `constexpr` some uninitialized value, which would be zero. i.e. `constexpr timestamp_t INVALID_TIMESTAMP = 0` and then tests could use that `if (m_time_start == INVALID_TIMESTAMP) { /*handle it*/ }`
Similar for other values.